resources/views/community/components/user/progression-status/root.blade.php

```diff
@@ -0,0 +1,88 @@
+<?php
+$widthsPreference = request()->cookie('progression_status_widths_preference');
+$widthMode = $widthsPreference;
+if ($widthMode !== 'equal' && $widthMode !== 'dynamic') {
+    $widthMode = 'equal';
+}
+?>
+
+<h4 class="!leading-none mb-2">Progression Status</h4>
```

Comment:
Should the entire widget be hidden if the user has no play history?
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/433da5d7-4ffd-4a2d-bdc9-942c3eeda0dd)

Reply:
Yep, I missed this. Resolved in latest.